ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -361,6 +362,21 @@ pattern CardanoNodeToClientVersion7 =
       :* Nil
       )
 
+-- | The hard fork enabled, and the Shelley, Allegra, Mary and Alonzo eras enabled
+-- Using 'ShelleyNodeToClientVersion5' for the Shelley-based eras , which
+-- enables new queries.
```

Comment:
Would it make sense to refer to what these new queries are?

Reply:
I figured that since comments are particularly susceptible to bitrot, it would be best if I mention the specific queries in a single place only, in this case the definition of `ShelleyNodeToClientVersion5`.

But I you prefer, I'm happy to the specific query (`GetRewardInfoPools`) at the definition of `CardanoNodeToClientVersion8` as well.

Reply:
> I figured that since comments are particularly susceptible to bitrot, it would be best if I mention the specific queries in a single place only, in this case the definition of ShelleyNodeToClientVersion5.

Sure, that makes sense. I'm ok with the comment as is.